shift_detector/checks/lda_check.py

```diff
@@ -54,28 +63,130 @@ def run(self, store) -> Report:
             self.cols = list(col_names)
             col_names = self.cols
 
-        df1_embedded, df2_embedded = store[LdaEmbedding(n_topics=self.n_topics, n_iter=self.n_iter, lib=self.lib,
-                                                        random_state=self.random_state, cols=self.cols,
-                                                        trained_model=self.trained_model, stop_words=self.stop_words,
-                                                        max_features=self.max_features)]
+        if self.lib == 'sklearn':
+            df1_embedded, df2_embedded, topic_words, all_models, all_dtms, all_vecs = store[LdaEmbedding(
```

Comment:
This retrieval from the store may be moved out of the if. The only difference I see are the last variable names and they are assigned again in the next line anyway

Reply:
These need to be separate calls as the last two variables are different for each call. Regardless to prevent confusion like yours, I changed the lib value in the store call to the respective library strings